data/json/items/gun/308.json

```
@@ -337,6 +338,26 @@
     "modes": [ [ "DEFAULT", "semi-auto", 1 ] ],
     "default_mods": [ "recoil_stock", "cheek_pad", "bipod", "rifle_scope_high_end_mount", "match_trigger", "muzzle_brake" ]
   },
+  {
+    "id": "scar_17s",
+    "copy-from": "scar_h",
+    "looks_like": "modular_ar15",
+    "type": "GUN",
+    "name": { "str": "civilian FN SCAR-H" },
```

You instead need to put one of those nouns into this name.
```suggestion
    "name": { "str": "civilian FN SCAR-H rifle" },
```